### PR TITLE
Add recipe for pyscrap

### PR DESCRIPTION
**Pull request by npavlovikj**
Add recipe for pyscrap

### CODE REVIEW

File: recipes/pyscrap/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  sha256: {{ sha256 }}
+  patches:
+    - import.patch  # [py3k]
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
```
Comment (synapticarbors) [inline recipes/pyscrap/meta.yaml]: Are the dependencies beyond `python` and `setuptools` (or `pip` if you switch over to using that as described in another comment) needed to `build` the package? They seem like runtime-only dependencies, although I'm not sure if `2to3` makes them necessary.

File: recipes/pyscrap/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    - python
+    - setuptools
+    - simplejson
+    - lxml
+    - libxml2
```
Comment (synapticarbors) [inline recipes/pyscrap/meta.yaml]: Are `libxml2` and `libxslt` needed? I couldn't find them referenced anywhere in the source. They seem like they're just dependencies of `lxml`, which will be pulled in automatically.

File: recipes/pyscrap/build.sh [hunk 3]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+if [[ "${PY_VER}" =~ 3 ]]
+then
+    2to3 -w -n setup.py
```
Comment (synapticarbors) [inline recipes/pyscrap/build.sh]: @conda-forge/help-python -- is there any policy regarding applying `2to3` to get python3 compatible builds?

**Comment by CJ-Wright** [comment]
I think this needs a rebase?

File: recipes/pyscrap/build.sh (hunk 3 above)
Comment (npavlovikj) [reply recipes/pyscrap/build.sh]: Thanks for the feedback @synapticarbors ! I fixed the recipe as suggested. I still left  the "2to3" part though - but I can remove it and limit the recipe to only py27 if needed.

Comment (npavlovikj) [reply recipes/pyscrap/build.sh]: @synapticarbors , are there any updates on this?

**Comment by synapticarbors** [comment]
@conda-forge/staged-recipes do we have any general directions when it comes to using `2to3` to get a python3 build from a python package that has only really be released for python2?

**Comment by ocefpaf** [comment]
> @conda-forge/staged-recipes do we have any general directions when it comes to using 2to3 to get a python3 build from a python package that has only really be released for python2?

We don't have a policy on that. IMO it is like a "patch" that upstream won't take it. I'm +0.5 on accepting this as long as @npavlovikj is OK and it is well tested.

**Pull request merged**